app/assets/stylesheets/base/_colors.scss

```diff
@@ -4,7 +4,7 @@
 $primary-text-color: #000;
 $secondary-text-color: #6D0839;
 $highlight-text-color: #D0E799;
-$header-text-color: #fff;
+$header-text-color: #555;
```

Comment:
Let's change this to `#fff`! We'll adjust the opacity of the actual header in the CSS.